src/Response.php

```diff
@@ -198,6 +202,9 @@ public static function fromString($string)
         if ($response->statusCode === static::STATUS_CODE_100) {
             $next = array_shift($lines); // take next line
             $next = empty($next) ? array_shift($lines) : $next; // take next or skip if empty
+            if (null === $next) {
+                throw new RuntimeException('Invalid response content');
+            }
```

Comment:
This is new behavior and requires a corresponding test.

Reply:
Tests added